tests/unit/t_hkdf.c

```diff
@@ -196,10 +634,20 @@ struct hkdf_test_vectors tv[] = {
       .okm_len     = 42 },
 
     { .which_sha = AWS_CRYPTOSDK_NOSHA, .ikm = tv_6_ikm, .ikm_len = 10 },
+
+    { .which_sha   = AWS_CRYPTOSDK_SHA256,
+      .ikm         = tv_7_ikm,
+      .ikm_len     = 22,
+      .salt        = tv_7_salt,
+      .salt_len    = 13,
+      .info        = tv_7_info,
+      .info_len    = 10,
+      .okm_desired = tv_7_okm_desired,
+      .okm_len     = 8129 },
 };
 
 int test_hkdf() {
-    for (int i = 0; i < 7; i++) {
+    for (int i = 0; i < 8; i++) {
```

Comment:
Instead of ``8`` it is better to use ``sizeof(tv)/sizeof(struct hkdf_test_vectors)`` so that this will always run the right number of tests when more are added to the array. I would also probably rename the type to ``struct hkdf_test_vector`` since a single instance of the struct refers to a single test vector. Not blocking on these.